test/parallel/test-tls-delayed-attach.js

```diff
@@ -24,17 +24,17 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const tls = require('tls');
-const fs = require('fs');
 const net = require('net');
 
 const sent = 'hello world';
 let received = '';
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fixtures.readKey('/keys/agent1-key.pem'),
```

Comment:
This actually doesn't need `/keys/` because the `readKey` already knows where to look.